control_examples

```diff
@@ -0,0 +1,15 @@
+Package: ni-scan-engine-veristand-{veristand_version}-labview-examples
+Version: {nipkg_version}
+Architecture: windows_x64
+Maintainer: National Instruments <support@ni.com>
+XB-Plugin: file
+Description: Provides LabVIEW examples for the Scan Engine and EtherCAT custom device API for NI VeriStand {veristand_version}.
```

Comment:
```suggestion
Description: Provides LabVIEW examples for the Scan Engine and EtherCAT custom device for NI VeriStand {veristand_version}.
```

Match R+F.

Reply:
Ok. I considered this an improvement as it makes it clear the example is for the API. Maybe it is sufficient to leave it assumed. Either way, I am not committed to including the additional descriptive word.